src/platform/windows/window.rs

```diff
@@ -715,15 +715,39 @@ unsafe fn init(
         events_loop_proxy,
     };
 
-    if let Some(_) = attributes.fullscreen {
-        win.set_fullscreen(attributes.fullscreen);
+    if let Some(_) = &attributes.fullscreen {
+        win.set_fullscreen(attributes.fullscreen.clone());
         force_window_active(win.window.0);
     }
 
     if let Some(dimensions) = attributes.dimensions {
         win.set_inner_size(dimensions);
     }
 
+    if attributes.center_window && !&attributes.fullscreen.is_some(){
+        let window_size = win.get_outer_size();
+
+        let monitor;
```

Comment:
Haven't tested if this compiles, but how about something like this?
```
let monitor = match window_attrs.start_monitor {
    None => window.get_primary_monitor(),
    Some(m) => m.inner,
};
```

Reply:
ahh right, matching is so nice, I'm just too used to write ifs, coming from c#